### PR TITLE
fix(curriculum): fixed Incorrect pre-existing code in step 30 in RWD 22 curriculum registration form

### LINKED ISSUE #52029: RWD curriculum registration form  - Incorrect pre-existing code in step 30
### Describe the Issue

This step requires a person to add a checkbox to a form but there is already markup for a checkbox in pre-existing code.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-30

### Expected behavior

The pre-existing markup for the checkbox should not be there in pre-existing code.

### Screenshots

bug_step30_Registration_Form

### PR DESCRIPTION
**Pull request by shoaib-31**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #52029 

I have fixed the incorrect pre-written code in responsive web design 22 where it was told to implement a check box but i was present there already.

### CODE REVIEW

**Comment by lasjorg** [comment]
Can I get you to fix a few issues with the text as well?

Main text change:

> Add a `label` element. Inside the newly created `label` element add an `input` element and set the `type` attribute to `checkbox`. Make this `input` element `required` so users can not sign up without agreeing to the terms and conditions.
>
> Add an `id` and `for` attribute with the value `terms-and-conditions` to the elements for accessibility.

First test text change:

> You should add a `label` element after the third `fieldset` element.

**Comment by shoaib-31** [comment]
> Can I get you to fix a few issues with the text as well?
> 
> Main text change:
> 
> > Add a `label` element. Inside the newly created `label` element add an `input` element and set the `type` attribute to `checkbox`. Make this `input` element `required` so users can not sign up without agreeing to the terms and conditions.
> > Add an `id` and `for` attribute with the value `terms-and-conditions` to the elements for accessibility.
> 
> First test text change:
> 
> > You should add a `label` element after the third `fieldset` element.

@lasjorg I will be pleased to do it, I will add a commit by tommorow and let you know :smile:

**Comment by shoaib-31** [comment]
@lasjorg Take a look, I have fixed text also :smile: 
Screenshot from 2023-10-20 20-27-12

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ dashedName: step-30
 
 You need to confirm that the user has read the terms and conditions.
 
-Add `label` element after the third `fieldset`, and a `input` element with `type` attribute of `checkbox` inside the newly added `label` element. Make this `input` element `required` because users should not sign up without reading the terms and conditions.
+Add a `label` element. Inside the newly created `label` element add an `input` element and set the `type` attribute to `checkbox`. Make this `input` element `required` so users can not sign up without agreeing to the terms and conditions.
 
-Don't forget to add an `id` attribute of `terms-and-conditions` for accessibility.
+Add an `id` and `for` attribute with the value &nbsp; &nbsp; &nbsp; &nbsp; `terms-and-conditions` to the elements for accessibility.
```
Comment (lasjorg) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md:14]:
```suggestion
Add an `id` and `for` attribute with the value `terms-and-conditions` to the elements for accessibility.
```

You have some extra whitespace here.

**Review by lasjorg — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md (hunk 1 above)
Comment (shoaib-31) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md:14]: I did that on purpose, the `terms and conditions` block was splitting into half.

Comment (lasjorg) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md:14]: When the text will wrap to a new line is dependent on the screen width of the device. You can not predict that.

Here is how it looks on my screen.

challenge-text

Please remove the spaces.

Comment (shoaib-31) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md:14]: Thanks, you are right :+1:

**Review by lasjorg — APPROVED** [review]
LGTM.

Thank you for your contribution.

**Comment by shoaib-31** [comment]
@lasjorg Please merge this a PR.

**Comment by lasjorg** [comment]
I can't. Please review the required sections.

> Review required
> At least 2 approving reviews are required by reviewers with write access.

> Merging is blocked
> Merging can be performed automatically with 2 approving reviews.

**Comment by shoaib-31** [comment]
@lasjorg , do you know anybody whom I should tag to get it merged?

**Comment by shoaib-31** [comment]
@Sembauke Please review it.

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @shoaib-31. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**